logspout/start.sh

```diff
@@ -33,5 +36,7 @@ done
 export SYSLOG_STRUCTURED_DATA="$(echo ${audius_loggly_token} | base64 -d)@41058 ${tags}"
 echo SYSLOG_STRUCTURED_DATA=${SYSLOG_STRUCTURED_DATA}
 
-# start logspout and point it to Loggly
-/bin/logspout multiline+syslog+tcp://logs-01.loggly.com:514
+# start logspout if audius_loggly_disable is not set
+if [[ -z "$audius_loggly_disable" ]]; then
```

Comment:
I missed this in my first review, but my vote is for something more truthy, like:

```
if [[ -n "$audius_loggly_enabled" ]]; then
```

It makes it easier to reason about if something is `true` or `false`, rather if something is `not true` or `not false`.

Reply:
unfortunately this env var is passed in so many other places and set in so many configs it's basically impossible to change and maintain backwards compatibility

Reply:
like it's baked into our existing dockerfiles for content/discovery

Reply:
Ah ok, since it's a legacy envar that sounds good then!